multigen/pipes.py

```diff
@@ -10,6 +13,8 @@
 from diffusers import StableDiffusionControlNetPipeline, ControlNetModel, UniPCMultistepScheduler
 from diffusers import StableDiffusionControlNetInpaintPipeline, DDIMScheduler
 from diffusers.schedulers import KarrasDiffusionSchedulers
+
+from multigen.pipelines.masked_stable_diffusion_img2img import MaskedStableDiffusionImg2ImgPipeline
```

Comment:
Importing from `multigen` works only if it is installed as a library. Since we are inside this library, it is more universal to write `.` instead of `multigen.` here.

Reply:
Do you mean this?
```suggestion
from .pipelines.masked_stable_diffusion_img2img import MaskedStableDiffusionImg2ImgPipeline
```

Reply:
Importing multigen should work when it's in PYTHONPATH